src/components/com_tjcertificate/site/views/certificate/tmpl/default.php

```diff
@@ -14,15 +14,20 @@
 use Joomla\CMS\Factory;
 use Joomla\CMS\Router\Route;
 use Joomla\CMS\Language\Text;
+use Joomla\CMS\Uri\Uri;
+use Joomla\CMS\HTML\HTMLHelper;
+
+HTMLHelper::StyleSheet('components/com_tjcertificate/assets/font-awesome-4.1.0/css/font-awesome.min.css');
```

Comment:
We already have a media folder. Add all the Font awesome related files in itt

Reply:
done